ete3/clustering/clustvalidation.py

```diff
@@ -192,13 +193,4 @@ def square_euclidean_dist(v1,v2):
         raise ValueError("Cannot calculate values")
     return  distance/valids
 
-try:
-   from scipy import stats
-except ImportError:
-    try:
-        from . import stats
-        default_dist = spearman_dist
-    except ImportError:
-        default_dist = euclidean_dist
-else:
-    default_dist = spearman_dist
+default_dist = spearman_dist
```

Comment:
So make a conditional import that shows a message detailing what to do if scipy is not installed?
And make scipy an optional dependency again.